pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -534,13 +534,6 @@ def test_get_indexer_pyarrow(self, as_td):
         result2 = target.get_indexer(index)
         tm.assert_numpy_array_equal(result2, expected)
 
-    def test_get_indexer_date_objs(self):
```

Comment:
instead of removing the test entirely, test the new behavior, with a comment about the history/change

Reply:
(havent looked closely, but this likely applies to tests below)